src/app/components/Account/ConsensusAccountDetailsView.tsx

```diff
@@ -44,6 +48,12 @@ export const ConsensusAccountDetailsView: FC<ConsensusAccountDetailsViewProps> =
   if (isLoading) return <TextSkeleton numberOfRows={7} />
   if (isError || !account) return <CardEmptyState label={t('account.cantLoadDetails')} />
 
+  const transactionsLabel = account.stats.num_txns.toLocaleString()
+  const transactionsAnchor = `${RouteUtils.getAccountRoute(
```

Comment:
part of https://github.com/oasisprotocol/explorer/issues/1767 issue was to link to validator page from search results. now we will have some links to validator page but tx number to account when you search for validators.

Reply:
Ah, I didn't notice
- https://pr-2030.oasis-explorer.pages.dev/search?q=GateOmega shows it
- https://pr-2030.oasis-explorer.pages.dev/mainnet/consensus/validators/oasis1qzk6qlmgnq40cq2n3jfkw3307feqngt4gvksfml6 doesn't show it

![image](https://github.com/user-attachments/assets/5271fa6d-2998-4731-b000-7eaadea16eb2)